shell/components/nav/TopLevelMenu.vue

```diff
@@ -726,18 +755,153 @@ export default {
         }
       }
 
+      .clusters-all {
+        display: flex;
+        flex-direction: row-reverse;
+        margin-right: 16px;
+        margin-top: 10px;
+
+        span {
+          display: flex;
+          align-items: center;
+        }
+      }
+
       .clusters {
         overflow-y: auto;
-        overflow-x: hidden;
+        max-height: 35vh;
```

Comment:
This introduces a void between clusters and categories. I don't think it's needed anymore?

![image](https://github.com/rancher/dashboard/assets/18697775/fd76f144-286c-43af-b51f-bb5540f859e6)